lib/trade_tariff_backend.rb

```diff
@@ -169,6 +169,19 @@ def cached_models
       [Heading, Certificate, AdditionalCode, Footnote]
     end
 
+    def clearable_models
```

Comment:
At some stage - not now - I should clearly understand the mechanical differences between all of the other models that don't need their caches clearing and the ones that do. Mainly because we may have missed the more obscure models that need clearing and haven't been identified by the tests and because if we understand exactly what the difference is we can document and implement the TimeMachine in a way that means that Sequel recognises it needs to update it's SQL when the actual date is different from the one used to generate the previously cached SQL.